py-polars/src/lazyframe/visitor/nodes.rs

```diff
@@ -328,6 +329,7 @@ pub(crate) fn into_py(py: Python<'_>, plan: &IR) -> PyResult<PyObject> {
                         .map_err(|err| PyValueError::new_err(format!("{err:?}")))?;
                     ("csv", options, cloud_options).into_py(py)
                 },
+                #[cfg(feature = "parquet")]
```

Comment:
*question*: Does `feature="parquet" imply `feature="serde"` such that `serde_json` is available?

(Thanks for this, btw!)

Reply:
Nope, it doesn't, there's still various issues if you only compile with `--no-default-features --features parquet`.

But that's for a future follow-up PR. I want to tackle this in bits.